src/components/Table/TableStyles.tsx

```diff
@@ -231,10 +231,17 @@ export const cardStyle: GridStyle = {
   },
   rowHoverColor: "none",
   nonHeaderRowHoverCss: Css.bshHover.bcGray700.$,
-  levels: {
-    1: { rowIndent: 24 },
-    2: { rowIndent: 48 },
-  },
+  // this will allow having N amount of nested childs without having to define each level margin
+  levels: new Proxy({} as Record<string, { rowIndent: number }>, {
```

Comment:
Thought: The proxy is cute/creative, but if we end up blessing this, we should probably just update the `GridStyle.levels` type to be a union of `Record<string, { rowIndent }> | `(level: number) => { rowIndent }`.

We control the types, so no need to torture ourselves with fulfilling them in esoteric ways, vs. more directly declaring the behavior we want.